packages/medusa/src/models/product-variant.ts

```diff
@@ -264,4 +266,7 @@ export class ProductVariant extends SoftDeletableEntity {
  *     nullable: true
  *     type: object
  *     example: {car: "white"}
+ *   purchasable:
```

Comment:
suggestion(non-blocking): It would be a good DX to add the different cases to the OAS